src/components/Paragraph/README.md

```diff
@@ -1,61 +1,105 @@
-## Available Sizes
-
-### Large Paragraph
+A `Paragraph` is a self-contained unit of one or more coherent sentences related to a single topic or idea.
 
 ```jsx
 <Paragraph size="large">
 	Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut bibendum hendrerit
-	augue gravida hendrerit. Vestibulum ornare eget lacus cursus rhoncus. Cras nec
-	lectus in sem viverra volutpat. Fusce placerat nunc eu libero egestas, sit
-	amet condimentum turpis ornare. Sed hendrerit metus elit, ac sollicitudin enim
-	vulputate et. Ut fermentum venenatis auctor. Mauris augue augue, hendrerit
-	quis risus eget, imperdiet feugiat odio. Nullam porta sollicitudin ipsum eu
-	varius. Nam hendrerit eleifend sagittis. Maecenas in sodales libero, nec
-	ultrices ante. Ut et sem nibh. Vivamus fringilla pharetra justo aliquet
-	pharetra.
+	augue gravida hendrerit.
+</Paragraph>
+
+<Paragraph>
+	Vestibulum ornare eget lacus cursus rhoncus. Cras nec
+	lectus in sem viverra volutpat.
 </Paragraph>
 ```
 
-### Medium Paragraph
+## Usage
+
+Use a `Paragraph` for any body copy comprising of one or more full sentences.
```

Comment:
I wonder if we'll want to write components like `<Paragraph>` to make it clear they're components? 🤷‍♂

Reply:
We could! I'm not in love with the component names in backticks, so lemme try that.

Reply:
You'd likely still want to include the backticks to highlight that they're code, so it'd be:

```md
Use a `<Paragraph>` for any body copy comprising of one or more full sentences.
```

instead of:

```md
Use a `Paragraph` for any body copy comprising of one or more full sentences.
```